### DISCUSSION TITLE
Run TASM on MS-DOS

### DISCUSSION THREAD

**Discussion by arsalanses** [copy/v86#514] [Q&A] [answered]
How could I add these tools such as (TASM, TLink & Debug) to ms-dos image for assembly programming?

**Comment by copy** [comment, marked as answer]
Create a a floppy disk or iso (which you can do on Linux for example with `mkisofs`) with the files on it and copy them over. Unfortunately v86 currently doesn't support multiple disks (https://github.com/copy/v86/issues/480, https://github.com/copy/v86/issues/463), so you'll need to use qemu or similar to copy the files over.
